Todo/Todo.cs

```diff
@@ -0,0 +1,627 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Globalization;
+using System.Security.Permissions;
+using System.Threading;
+
+namespace To_Do_List
+{
+    public class To_do : IComparable<To_do>
+    {
+        public int Todo_ID { get; set; }
+        public DateTime date { get; set; }
+        public string Task { get; set; }
+        public int Lvl_Imp { get; set; }
+
+        public To_do(int Todo_ID, DateTime date, string Task, int Lvl_Imp)
+        {
+            this.Todo_ID = Todo_ID;
+            this.date = date;
+            this.Task = Task;
+            this.Lvl_Imp = Lvl_Imp;
+        }
+        public int CompareTo(To_do other)
+        {
+            return this.Todo_ID.CompareTo(other.Todo_ID);
+        }
+
+    }
+
+
+    class Program
+    {
+        //commonly used string tabbing
+        public static string mt = "\t\t\t";
+        public static string st = "\t\t ";
+        public static string xst = "\t\t";
+
+        //UI   
+        public static void header()  //Draw the main Menu Header
+        {
+            Console.Clear();
+            DateTime dtu = DateTime.Now;
+
+            Console.WriteLine("\n\n\n\t\t\t\t\t" + dtu.ToString("MM-dd-yyyy"));
+            Console.WriteLine("\t\t====================================");
+            Console.WriteLine("\t\t\t TO-DO List");
+            Console.WriteLine("\t\t====================================");
+        }
+
+        public static void footer()  //Draw the main Menu Footer
+        {
+            Console.WriteLine("\t\t====================================");
+        }
+
+
+
+        public static void Prompt(String msg)  //Deliver common error messages and instructions
+        {
+            header();
+            Console.WriteLine("\n\n" + st + msg + "\n\n");
+            footer();
+            Console.Write(st + "Press <any> key to continue:");
+            Console.ReadKey();
+
+        }
+
+
+        public static bool chk_date(String dateChk)  //checking for a valid date entry
+        {
+            string[] formats = { "M-d-yyyy" };
+            DateTime parsedDateTime;
+            if (DateTime.TryParseExact(dateChk, formats, new CultureInfo("en-US"),
+                    DateTimeStyles.None, out parsedDateTime))
+            {
+                return true;
+            }
+            else
+            {
+                return false;
+            }
+        }
+
+        public static void Main()
+        {
+            Random rnd = new Random();
+            int ID = rnd.Next(89);  //generate random ID
+
+            List<To_do> TD_Task = new List<To_do>();  //setup the list container, just storing to memory fix this
```

Comment:
Part of the assignment was to make the data persist in a SQL lite database so your items would be saved after the program quits, and you would able to continue using the list.